lib/tasks/fix_parent_child.rake

```diff
@@ -0,0 +1,288 @@
+namespace :scihist do
+  namespace :data_fixes do
+    desc "Genre adjustments, per https://github.com/sciencehistory/scihist_digicoll/issues/1275"
+    task :fix_parent_child => :environment do
+      adjustments = {
```

Comment:
I would have called this something descriptive instead of "adjustments", like `child_to_parent` or something. But that's not going to stop me from approving.